package.json

```diff
@@ -17,7 +17,7 @@
     "test": "mocha -R spec -t 16000 test/*-test.js"
   },
   "dependencies": {
-    "cheerio": "~0.19.0",
+    "entities": "^1.1.1",
```

Comment:
Updated. It'll be better for merge, also take less size in the browserify bundle, so it's fine.